src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/VariableDeclarationUsageDistanceCheck.java

```diff
@@ -152,14 +173,15 @@
  *  - to check the final variables;
  * </p>
  * <pre>
- * &lt;module name="VariableDeclarationUsageDistance"&gt;
- *     &lt;property name="allowedDistance" value="4"/&gt;
- *     &lt;property name="ignoreVariablePattern" value="^temp.*"/&gt;
- *     &lt;property name="validateBetweenScopes" value="true"/&gt;
- *     &lt;property name="ignoreFinal" value="false"/&gt;
+ * &lt;module name=&quot;VariableDeclarationUsageDistance&quot;&gt;
```

Comment:
@pbludov I believe you are doing these type of changes by hand, correct? I see the xdoc still uses `"` instead of `&quot;`.

Maybe consider enforcing the program to replace these for you so it won't be a manual process and future contributors will be forced to make this change too.
Doing something like this on xdoc will probably require something like `XdocPagesTest.testAllXmlExamples` (or a modification to it).